scripts/tests/unit/test_schema_cleaner.py

```diff
@@ -262,6 +262,20 @@ def test_multiline_short_description_raises(self):
         with self.assertRaisesRegex(ValueError, 'single line'):
             cleaner.single_line_short_description(schema)
 
+    def test_very_long_short_description_warns_strict_disabled(self):
+        schema = {'field_details': {
+            'name': 'fake_schema',
+            'short': "Single line but really long. " * 10}}
+        with self.assertWarnsRegex(UserWarning, 'under 120 characters \(current length: 290\)'):
+            cleaner.single_line_short_description(schema, strict_mode_enabled=False)
```

Comment:
Thanks for adding the tests.

Could you also wrap that in a block that ensures no exception is raised?

Reply:
Something like below?

```python
try:
    with self.assertWarnsRegex(UserWarning, 'under 120 characters \(current length: 290\)'):
        cleaner.single_line_short_description(schema, strict_mode_enabled=False)
except Exception:
    self.fail("cleaner.single_line_short_description() raised Exception unexpectedly."
```

Reply:
Yeah I looked for an assertion to ensure the absence of an exception, and haven't found one. So this works 👍